website/src/models/group.js

```diff
@@ -475,11 +475,6 @@ GroupSchema.methods.leave = function(user, keep, mainCb){
 
 GroupSchema.methods.toJSON = function() {
   var doc = this.toObject();
-  if(doc.chat){
```

Comment:
I elected to do this at the api level instead of here. The reason being, if someone used the `GET /groups/id/chat` route, the flags object was still coming through.